docs/DeveloperGuide.md

```diff
@@ -297,6 +297,8 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user                                       | mark applicants as "Done"                     | know which applicants I have attended to                                  |
 | `* * *`  | user                                       | unmark applicants from "Done" to "Not Done"   | reflect the correct current status I have with the applicant              |
 | `* * *`  | user                                       | save applicant data                           | refer to it in the future without having to type in their data again      |
+| `* *`    | regular user                               | delete all applicants that are marked as done | so that I can focus only on applicants that I have not worked on, and easily reduce clutter in the address book |
+| `* *`    | regular user                               | delete multiple applicants at once            | so that I can delete applicants more efficiently                          |
```

Comment:
Maybe you can move this up to go with the other delete user story? Or maybe the two can be combined into one? It's up to you to choose what you think is best! 🤣